docs/src/architecture/secureboot.md

```diff
@@ -0,0 +1,50 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Secure Boot
+
+This section describes secure boot and how to create secure keys.
```

Comment:
Should we add after one empty line something like that:

The reader is expected to know the fundamentals of UEFI and have a basic understanding of Secure Boot [UEFI specification](https://uefi.org/specifications).

?